### PR TITLE
Fixes #21396 remove using 'user.jpg' file'

### PR DESCRIPTION
**Pull request by amirfefer**

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -384,8 +384,8 @@ def gravatar_image(email)
   end
 
   def gravatar_html_options
-    { :onerror => "this.src='#{path_to_image('user.jpg')}'",
-      :alt     => _('Change your avatar at gravatar.com') }
+    { :onerror => "$(this).addClass('fa fa-user')",
+      :alt     => '' }
```
Comment (ohadlevy) [inline app/helpers/application_helper.rb:388]: why drop the comment? if gravatar is enabled users should know how to use it?

Comment (amirfefer) [reply app/helpers/application_helper.rb:388]: alternative text renders when error occurred while loading the image.
Now we're using icon instead, so we can use `title`attribute instead.
anyway, that text wasn't been visible for users before

Comment (ohadlevy) [reply app/helpers/application_helper.rb:388]: ACK - how hard would it be to write a test for this?

**Comment by ohadlevy** [comment]
test failure unrelated, thanks @amirfefer

**Pull request merged**